### PR TITLE
updated sha256 and url to release version 3.2.0

### PR DESCRIPTION
**Pull request by shimwell**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by shimwell** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+{% set version = "3.2.0" %}
+{% set sha256 = "1438179e9d1c0807af4fbabc63efda45b1c995e2e568450bc6ae5ca35b18ca8f" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://github.com/svalinn/DAGMC/archive/v{{ version }}-rc.tar.gz
+  url: https://github.com/svalinn/DAGMC/archive/v{{ version }}.tar.gz
```
Comment (bam241) [inline recipe/meta.yaml]:
```suggestion
  url: https://github.com/svalinn/DAGMC/archive/{{ version }}.tar.gz
```

the `v` disappeared from the URL...

**Comment by shimwell** [comment]
Looks like Eigen 3 is not being found

```Could not find a package configuration file provided by "Eigen3" with any ...```

Looks like Eigen is available on conda-forge so I guess this can be added 
https://github.com/conda-forge/eigen-feedstock

Some packages are in host but not in the run section so I shall try this Eigen in the same way

**Comment by shimwell** [comment]
What do people think about including Embree (which is on conda-forge) and double down in this Dagmc build?

**Comment by shimwell** [comment]
Also I am wondering if this line can be removed

https://github.com/conda-forge/dagmc-feedstock/blob/3376e7cf235e5baa13f841ec40479ec3073fabc3/recipe/meta.yaml#L22

It appears the MOAB conda build doesn't have fortran

https://github.com/conda-forge/moab-feedstock/blob/7b8baf10019cfddb457f6158782d8c7dc1beb10d/recipe/build.sh#L23

**Comment by bam241** [comment]
> What do people think about including Embree (which is on conda-forge) and double down in this Dagmc build?

I don't know my understanding is that `Embree` was still experimental so I would not, maybe @pshriwise have some thoughts on that point ?

**Comment by bam241** [comment]
> Also I am wondering if this line can be removed
> 
> https://github.com/conda-forge/dagmc-feedstock/blob/3376e7cf235e5baa13f841ec40479ec3073fabc3/recipe/meta.yaml#L22
> 
> It appears the MOAB conda build doesn't have fortran
> 
> https://github.com/conda-forge/moab-feedstock/blob/7b8baf10019cfddb457f6158782d8c7dc1beb10d/recipe/build.sh#L23

I'd say if `dagmc` does not depends on it directly we should not include it. if `MOAB` needs it they should include it in their build.
-> so I agree with you this should probably go away...

**Comment by shimwell** [comment]
Similar question with these two lines in the build host section
https://github.com/conda-forge/dagmc-feedstock/blob/3376e7cf235e5baa13f841ec40479ec3073fabc3/recipe/meta.yaml#L26-L27

They appear to be included in the Moab meta.yaml so perhaps they are not needed in the DAGMC meta.yml

https://github.com/conda-forge/moab-feedstock/blob/7b8baf10019cfddb457f6158782d8c7dc1beb10d/recipe/meta.yaml#L75-L76

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
 requirements:
   build:
     - cmake
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler("fortran") }}
   host:
     - hdf5
     - moab 5.*
     - libblas
```
Comment (shimwell) [inline recipe/meta.yaml]:
```suggestion
```

Comment (Shimwell) [inline recipe/meta.yaml:25]:
```suggestion
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,30 +1,30 @@
...
     - liblapack
```
Comment (shimwell) [inline recipe/meta.yaml]:
```suggestion
```

**Comment by shimwell** [comment]
Appears to be happily building without liblapac and libblas

**Review by gonuke — CHANGES_REQUESTED** [review]
Thanks @Shimwell!

Your changes look ok to me, but we might be building the wrong mix of DAGMC pieces. Do we need to tweak the CMake line in the build script? E.g. Turn off DAGTally? (Perhaps we're ok with this build line?)

**Comment by shimwell** [comment]
> Thanks @Shimwell!
> 
> Your changes look ok to me, but we might be building the wrong mix of DAGMC pieces. Do we need to tweak the CMake line in the build script? E.g. Turn off DAGTally? (Perhaps we're ok with this build line?)

I have now added all the build args with default settings. The only non default is DBUILD_CI_TESTS which has been set to ON as I think this could be useful in the test section of the meta.yml

**Comment by shimwell** [comment]
I believe the tests are failing as ```make_watertight: command not found```

**Comment by gonuke** [comment]
I think that moving the build script to a sub directory caused nothing to build

**Comment by bam241** [comment]
just wondering is everything installed at the correct location ?

**Comment by gonuke** [comment]
The `build.sh` script that conda-forge is trying to use is in the wrong place.  It doesn't find `build.sh` and does nothing - no failure, because it just thinks the build process is null.  But then when it runs tests it thinks that make watertight is there.

File: recipe/build.sh [hunk 4]
```diff
@@ -1,7 +1,32 @@
 #!/usr/bin/env bash
 
 # Install DAGMC
-cmake -DMOAB_DIR="${PREFIX}" \
+# default options from
+# https://github.com/svalinn/DAGMC/blob/develop/cmake/DAGMC_macros.cmake
+
+cmake -DBUILD_MCNP5=OFF
```
Comment (gonuke) [inline recipe/build.sh]: each of these lines needs a backslash ("\") to continue the line

Comment (shimwell) [reply recipe/build.sh]: Thanks sorry about that, looks like that might have fixed it

**Review by gonuke — APPROVED** [review]
LGTM - thanks @Shimwell

**Pull request merged**